### PR TITLE
Pass command-line arguments

### PR DESCRIPTION
**Pull request by evalsocket**
# TL;DR
Pass command-line arguments

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Tracking Issue
https://github.com/lyft/flyte/issues/167

### CODE REVIEW

File: cmd/controller/cmd/root.go [hunk 1]
```diff
@@ -97,12 +97,14 @@ func init() {
...
 }
 
-func initConfig(_ *cobra.Command, _ []string) error {
+func initConfig(cmd *cobra.Command, _ []string) error {
 	configAccessor = viper.NewAccessor(config.Options{
 		StrictMode:  true,
 		SearchPaths: []string{cfgFile},
 	})
 
+	configAccessor.InitializePflags(cmd.Flags())
```
Comment (kumare3) [inline cmd/controller/cmd/root.go:106]: isnt this already initialized - https://github.com/lyft/flytectl/blob/master/cmd/get/task.go#L62

Comment (evalsocket) [reply cmd/controller/cmd/root.go:106]: it is a different issue

Comment (evalsocket) [reply cmd/controller/cmd/root.go:106]: @kumare3 any update on this pr

**Review by EngHabu — APPROVED** [review]

**Comment by evalsocket** [comment]
@kumare3 can you review the PR?

**Comment by EngHabu** [comment]
Hey @evalsocket, can you rebase and make sure tests are green?

**Comment by evalsocket** [comment]
@EngHabu pr is ready

**Review by EngHabu — APPROVED** [review]
Thank you!

**Review by anandswaminathan — APPROVED** [review]

**Pull request merged**